stdlib/public/core/SetBridging.swift

```diff
@@ -412,17 +412,34 @@ extension _CocoaSet {
   @usableFromInline
   internal struct Index {
     @usableFromInline
-    internal var storage: Storage
+    internal var _object: Builtin.BridgeObject
+    @usableFromInline
```

Comment:
I don't think these need to be `usableFromInline` anymore, do they?

(Though if you do remove it, please do it in a follow-up. We need to get those optimize-test bots back sooner rather than later.)